examples/admin_create_topic/admin_create_topic.go

```diff
@@ -0,0 +1,93 @@
+// Create topic
+package main
+
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+	"context"
+	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+
+	if len(os.Args) != 5 {
+		fmt.Fprintf(os.Stderr,
+			"Usage: %s <broker> <topic> <partition-count> <replication-factor>\n",
+			os.Args[0])
+		os.Exit(1)
+	}
+
+	broker := os.Args[1]
+	topic := os.Args[2]
+	numParts, err := strconv.Atoi(os.Args[3])
+	if err != nil {
+		fmt.Printf("Invalid partition count: %s: %v\n", os.Args[3], err)
+		os.Exit(1)
+	}
+	replicationFactor, err := strconv.Atoi(os.Args[4])
+	if err != nil {
+		fmt.Printf("Invalid replication factor: %s: %v\n", os.Args[4], err)
+		os.Exit(1)
+	}
+
+	// Create a new AdminClient.
+	// AdminClient can also be instantiated using an existing
+	// Producer or Consumer instance, see NewAdminClientFromProducer and
+	// NewAdminClientFromConsumer.
+	a, err := kafka.NewAdminClient(&kafka.ConfigMap{"bootstrap.servers": broker})
+	if err != nil {
+		fmt.Printf("Failed to create Admin client: %s\n", err)
+		os.Exit(1)
+	}
+
+	// Contexts are used to abort or limit the amount of time
+	// the Admin call blocks waiting for a result.
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Create topics on cluster.
+	// Set Admin options to wait for the operation to finish (or at most 60s)
+	maxDur, err := time.ParseDuration("60s")
+	if err != nil {
```

Comment:
i wouldn't check for / handle bugs in standard go libraries

Reply:
Right, my intention was for this example to be a nice cut'n'paste template for our users, which might want to pass the timeout as user input.
And there is no harm in a bit of extra error checking

Reply:
there's harm in that it makes the example longer and some people may 'learn' that they should do this in this scenario. my opinion is still the other way, but minor detail.

Reply:
I disagree, generally people should check the error value, it is good practice.

As for example length, we could skip this option completely in this example to save some lines, but I believe a proper example of an option is important.